crates/meroctl/src/cli/call.rs

```diff
@@ -25,17 +26,17 @@ pub const EXAMPLES: &str = r"
     EXAMPLES
 ))]
 pub struct CallCommand {
-    #[arg(value_name = "CONTEXT_ID", help = "ContextId of the context")]
-    pub context_id: ContextId,
+    #[arg(value_name = "CONTEXT_ID", help = "ContextId or alias of the context")]
+    pub context_id: String,
```

Comment:
anywhere you have `_id` that could also be a context, shouldn't anymore.

`--context` is more appropriate